.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "cSpell.words": ["instuctions", "Stavok"]
+  "cSpell.words": ["fiest", "instuctions", "Stavok", "webfonts"]
```

Comment:
Why are we including `instuctions` here? Shouldn't we want it to notice that and ask us to change it to `instructions`?

Reply:
done!